### PR TITLE
[WIP] Version update

### PR DESCRIPTION
**Pull request by rawkode**
Updating to latest release of faas 0.6.6b

### CODE REVIEW

**Comment by alexellis** [comment]
Have you also seen the extended YAML file and the code under sample_functions/?

**Comment by rawkode** [comment]
@alexellis Hopefully that's everything

File: docker-compose.extended.yml [hunk 1]
```diff
@@ -87,15 +87,15 @@ services:
...
+                constraints:
                     - 'node.role == manager'
                     - 'node.platform.os == linux'
 
     # Sample functions go here.
 
     # Pass a username as an argument to find how many images user has pushed to Docker Hub.
     hubstats:
-        image: alexellis2/faas-dockerhubstats:latest
+        image: alexellis2/faas-dockerhubstats:latest0d0cc708600eb671a314af802d71e0417369311c
```
Comment (alexellis) [inline docker-compose.extended.yml]: ? intentional?

File: sample-functions/CHelloWorld/src/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.6 as builder
 RUN apk add --no-cache curl \
-    && curl -SL https://github.com/openfaas/faas/releases/download/0.6.4/fwatchdog > /usr/bin/fwatchdog \
+    && curl -SL https://github.com/openfaas/faas/releases/download/0.6.6b/fwatchdog > /usr/bin/fwatchdog \
```
Comment (alexellis) [inline sample-functions/CHelloWorld/src/Dockerfile]: There must be more examples of pulling the watchdog in..?

**Comment by alexellis** [comment]
Closing as out of date / unresolved comments. Please feel free to raise a new PR to update to latest images.

**Pull request closed without merging**